pkg/cmd/trigger.go

```diff
@@ -65,7 +66,9 @@ needed to create the triggered event.
   payment_intent.created
   payment_intent.payment_failed
   payment_intent.succeeded
-  payment_method.attached`,
+  payment_method.attached
+
+  --event <evt_id>`,
```

Comment:
We should improve the help message here. Something like this:
```

You can also resend a past event using the --event flag:
  stripe trigger --event evt_123
```

Reply:
I'll improve the help messages, but for the livemode I'll do that in a later PR.